pkg/util/sysctl/sysctl.go

```diff
@@ -56,6 +56,9 @@ type procSysctl struct {
 func (*procSysctl) GetSysctl(sysctl string) (string, error) {
 	data, err := os.ReadFile(path.Join(sysctlBase, sysctl))
 	if err != nil {
+		if os.IsNotExist(err) {
+			return "0", nil
+		}
```

Comment:
I think this check is better done at the caller level. E.g. here:

https://github.com/kubevirt/kubevirt/blob/c7b99dbc1a585844d1e90f44e22c04c746f60e40/pkg/network/driver/nmstate/status.go#L234-L237

Doing that in `GetSysctl` will make all the calls ignore `ErrNotExist`. This is probably not desirable.

Reply:
ok , i modify to  only check  IPv6GetForwarding